src/DynamoCoreWpf/Commands/WorkspaceCommands.cs

```diff
@@ -27,6 +27,7 @@ public partial class WorkspaceViewModel
         private DelegateCommand _unpauseVisualizationManagerUpdateCommand;
         private DelegateCommand _showHideAllGeometryPreviewCommand;
         private DelegateCommand _showInCanvasSearchCommand;
+        private DelegateCommand _showNodeAutoCompleteSearchCommand;
```

Comment:
Can we rename these private fields to not have the C++ style of naming them?

Reply:
Updated in latest commit